### PR TITLE
[bugfix] Fix issue when optional attributes are not provided in cloudCtxProfile client code.

### PR DESCRIPTION
**Pull request by lhercot**

### CODE REVIEW

File: client/cloudCtxProfile_service.go [hunk 1]
```diff
@@ -13,90 +13,105 @@ func (sm *ServiceManager) CreateCloudContextProfile(name string, tenant string,
...
-							"cloudProviderId": "%s"
+		`, vrf))
+		vrfCon, err := container.ParseJSON(vrfAttach)
+		if err != nil {
+			return nil, err
+		}
+		jsonPayload.ArrayAppend(vrfCon.Data(), cloudCtxProfile.ClassName, "children")
+	}
+
+	// Optional Attributes
```
Comment (akinross) [inline client/cloudCtxProfile_service.go:73]: is this comment needed?

File: client/cloudCtxProfile_service.go [hunk 2]
```diff
@@ -13,90 +13,105 @@ func (sm *ServiceManager) CreateCloudContextProfile(name string, tenant string,
...
+	jsonPayload.Array(cloudCtxProfile.ClassName, "children")
 
-	cidrJSON := []byte(fmt.Sprintf(`
-	{
-		"cloudCidr": {
-			"attributes": {
-				"addr": "%s",
-				"primary": "yes",
-				"status": "created,modified"
+	// Required Attributes
```
Comment (akinross) [inline client/cloudCtxProfile_service.go:18]: should this be changed to optional?

File: client/cloudCtxProfile_service.go (hunk 1 above)
Comment (lhercot) [reply client/cloudCtxProfile_service.go]: removed

File: client/cloudCtxProfile_service.go (hunk 2 above)
Comment (lhercot) [reply client/cloudCtxProfile_service.go]: removed the comment

**Pull request merged**